src/control/lib/control/response.go

```diff
@@ -1,5 +1,5 @@
 //
-// (C) Copyright 2020-2021 Intel Corporation.
+// (C) Copyright 2020-2022 Intel Corporation.
```

Comment:
Hmm. Just realized that I forgot to set up my pre-commit hook in the new clone I'm using for el8 development. Fixed.